app/results.tsx

```diff
@@ -207,52 +239,65 @@ export function Results() {
             <GithubIcon />
           </a>
         </div>
-        <input
-          autoFocus
-          type="search"
-          value={q}
-          onChange={(e) => setQ(e.target.value)}
-          className="w-[400px] ring-violet-500 focus:ring-1 outline-none max-w-full bg-violet-50 border border-violet-200 text-violet-900 text-sm rounded focus:border-violet-300 block p-2 dark:bg-violet-950 dark:border-violet-600 dark:placeholder-violet-400 dark:text-violet-300"
-          placeholder={`Search ${
-            results.length ? `${results.length} ` : ""
-          }channels`}
-        />
+        <div className="flex no-wrap">
+          <input
+            autoFocus
+            type="search"
+            value={q}
+            onChange={(e) => setQ(e.target.value)}
+            className="w-[400px] ring-violet-500 focus:ring-1 outline-none max-w-full bg-violet-50 border border-violet-200 text-violet-900 text-sm rounded focus:border-violet-300 block p-2 dark:bg-violet-950 dark:border-violet-600 dark:placeholder-violet-400 dark:text-violet-300"
```

Comment:
can we make this have className `cursor-pointer`?

Also is there a reason you're using a `Dropdown` here instead of a `Select`? seems more fitting to use a Select.